pages/events/seminar-series.vue

```diff
@@ -42,7 +42,7 @@
               v-for="({ component, styles, data }, elementIndex) in row"
               :key="`${elementIndex}`"
             >
-              <AppCta
+              <UiAppCta
```

Comment:
Severals `v-bind` on this page

Reply:
Changed in https://github.com/Qiskit/qiskit.org/pull/2980/commits/eef6c40a534c38b7a1e73640346d6f0071d25be8.

There is one that is not related to the "UiAppCta" component. I left it because it would involve a little bit more unrelated work and it might be better to do it when we update that component.